app/views/admin/attachments/edit.html.erb

```diff
@@ -1,9 +1,6 @@
 <% content_for :page_title, "Editing #{attachment.attachable_model_name} #{attachment.readable_type} attachment" %>
 <% content_for :title, "Edit #{attachment.readable_type} attachment" %>
 <% content_for :context, "Attachments for #{attachment.attachable_model_name}" %>
-<% content_for :back_link do %>
-  <%= render "govuk_publishing_components/components/back_link", href: attachable_attachments_path(attachable) %>
-<% end %>
```

Comment:
The [forms](https://github.dev/alphagov/whitehall/blob/ad6e0737bd5191efae51a96ed2fa9ddf615a54d8/app/views/admin/attachments/_form.html.erb) partial doesn't seem to have a cancel button, do we need to update that aswell?

Reply:
Good catch, it was removed here:
https://github.com/alphagov/whitehall/commit/ff67c0b95d575b0787934d0e8882707597e26d9e#diff-747aa398dc59115a77970d9f5e8e860d990f10cd6b79c1482f36444b8d31bfddL69

Have added it back in.